Mlem/Views/Tabs/Profile/User View.swift

```diff
@@ -122,6 +128,10 @@ struct UserView: View {
         .headerProminence(.standard)
         .refreshable {
             await tryLoadUser()
```

Comment:
Github flags this as dark red. I think `try await` is more common--not sure if there's a functional difference

Reply:
`try` here is part of the name of the method and _not_ the keyword `try`.  Is that frowned upon?

Reply:
Oh I see. Yeah that's fine--maybe not the *clearest* possible name but not confusing enough to be worth changing.